examples/ray/batch-split-proxy/training/requirements.txt

```diff
@@ -0,0 +1,7 @@
+alibi==0.5.4
```

Comment:
I believe the current master uses 0.5.5

Reply:
This is only used to download training dataset.

Reply:
Ok should be fine